src/pybind/mgr/dashboard/frontend/src/app/shared/components/date-time-picker/date-time-picker.component.html

```diff
@@ -1,16 +1,18 @@
-<div cdsCol
-     class="form-item">
+<div cdsCol>
```

Comment:
Since we have two or more than 2 items side by side, let's add the class `form-item-append` so it won't break the styles in other pages like the snapshedule form or rbd move to trash expiration.
```suggestion
<div cdsCol
        class="form-item-append">
```

Reply:
changes are done